system_tests/datastore.py

```diff
@@ -271,7 +287,13 @@ def test_projection_query(self):
         self.assertEqual(catelyn_stark_dict,
                          {'name': 'Catelyn', 'family': 'Stark'})
 
-        catelyn_tully_entity = entities[3]
+        if EMULATOR_DATASET is None:
```

Comment:
1. I will add the order to the query
1. Is there a way to add indexes to the emulator?

Thanks!